cmd/interchaincmd/relayercmd/start.go

```diff
@@ -50,7 +57,16 @@ func start(_ *cobra.Command, _ []string) error {
 		return err
 	}
 	switch {
-	case network.Kind == models.Local || network.Kind == models.Fuji:
+	case network.ClusterName != "":
```

Comment:
is this check is enough to decide if cluster is remote and ssh needed?
Do we need to also load ClusterConfig and check `clusterConf.Local` ?

Reply:
pls ignore this comment. This is code that was tested already and just moved  within code